ui.tests/test-module/specs/fileinput/fileinput.runtime.spec.js

```diff
@@ -0,0 +1,117 @@
+        expect(formContainer._model.items.length, "model and view elements match").to.equal(Object.keys(formContainer._fields).length);
+        Object.entries(formContainer._fields).forEach(([id, field]) => {
+            expect(field.getId()).to.equal(id)
+            expect(formContainer._model.getElement(id), `model and view are in sync`).to.equal(field.getModel())
+            checkHTML(id, field.getModel().getState())
+        });
+    })
+
+    it(" model's changes are reflected in the html ", () => {
+        Object.entries(formContainer._fields).forEach(([id, field]) => {
+            let model = field.getModel();
+            model.value = {
+                "name" : "abc",
+                "size" : 0,
+                "mediaType" : "application/octet-stream",
+                "data" : "http://xyz.com/def.pdf"
+            }
+            checkHTML(model.id, model.getState()).then(() => {
+                model.visible = false
+                return checkHTML(model.id, model.getState())
+            }).then(() => {
+                model.enabled = false
+                return checkHTML(model.id, model.getState())
+            })
+        });
+    });
+
+    it(" html changes are reflected in model ", () => {
+        Object.entries(formContainer._fields).forEach(([id, field]) => {
+            let model = field.getModel();
+            let fileName = 'empty.pdf';
+            if (model.visible && model.enabled) {
+                cy.get(`#${id}`).find("input").attachFile(fileName).then(x => {
+                    let expectedFileName = Array.isArray(model.getState().value) ? model.getState().value[0].name : model.getState().value.name;
+                    expect(expectedFileName).to.equal(fileName)
+                })
+            }
+        });
+        cy.window().then(function(win) {
+            win.guideBridge.getFormDataObject({
+                "success" : function (resultObj) {
+                    let afFormData = resultObj.data;
+                    expect(afFormData).to.exist // "af form data object not present in guideBridge#getFormDataObject API");
+                    expect(afFormData.data).to.exist // "form data not present in guideBridge#getFormDataObject API");
+                    expect(afFormData.contentType).to.equal("application/json");
+                    assert.equal(afFormData.attachments.length, 2, "incorrect attachments returned from guideBridge#getFormDataObject API");
+                    afFormData.attachments.forEach(function(attachment) {
+                        expect(attachment.dataRef).to.contain('fileinput');
+                    });
+                    // explicitly calling this to ensure there are no errors during this API invocation
+                    let htmlFormData = afFormData.toHTMLFormData();
+                    // Display the key/value pairs
+                    for (let pair of htmlFormData.entries()) {
+                        console.log(pair[0]+ ', ' + pair[1]);
```

Comment:
is there any use of doing console.log in the tests, why not assert it?

Reply:
We can't assert, I have added a comment, on why we do console log. This is just to ensure there is no error